test/unit/org/apache/cassandra/repair/FuzzTestBase.java

```diff
@@ -540,27 +528,11 @@ private static RepairOption repairOption(RandomSource rs, Cluster.Node coordinat
     {
         RepairType type = repairTypeGen.next(rs);
         PreviewType previewType = previewTypeGen.next(rs);
-        boolean accordRepair = type == RepairType.FULL && previewType == PreviewType.NONE ? rs.nextBoolean() : false;
         List<String> args = new ArrayList<>();
         args.add(ks);
         List<String> tables = tablesGen.next(rs);
         args.addAll(tables);
-        if (accordRepair)
```

Comment:
this was adding non-deterministic behavior to repair tests.  I spoke with Ariel about this and also pointed out that the new accord logic is blocking (repair expects async)... I didn't want to tackle fixing this logic so just removing this for now to keep CI stable...

we will need to fix the accord repair logic as it can cause perf regressions for repair